tables/el.ctb

```diff
@@ -1,6 +1,13 @@
-# LibLouis Table for Greek Braille - Common Rules
+# LibLouis table for both modern and ancient Greek braille
 # ελληνικό σύστημα μπράιγ [Greek Braille System]
 # κέντρο εκπαίδευσης και αποκατάστασης τυφλών [Center for Education and Rehabilitation for the Blind]
+#
+# -----------
+#-name: ελληνικό σύστημα μπράιγ [Greek Braille System]
+#-display-name: Greek (modern and ancient)
```

Comment:
Actually I think the info "modern and ancient" is not relevant for the user. If it would say only "modern" or only "ancient", it would be relevant, but if there is no choice I think it's better to drop it. Of course this is under the assumption that "ancient" is removed from the name of gr-bb.ctb, as you have done in the other PR.

Of course it can stay in the description on the first line, but the "name" metadata is what is presented to the user in the list of available tables.